### ISSUE TITLE
Publish Server Core image with da-DK culture

### ISSUE THREAD

**Issue by RasEza** [microsoft/Windows-Containers#95]
Windows Server Core images with various locales are already published at https://mcr.microsoft.com/v2/windows/servercore/tags/list. 
To better support applications that are strongly tied to the da-DK locale and language pack, I request that a Windows Server Core image be published with this configuration.

**Comment by weijuans-msft** [comment]
@RasEza Can you share more details why da-DK locale and language pack are needed? What are the use cases? What apps? How many customers do you have? Where do you run your containers? We did publish locales and language packs early on a few years back but we didn't see huge demand from customers, hence we prioritized to invest on other customer asks.

**Comment by weijuans-msft** [comment]
@RasEza gentle ping. I am trying to do a better job on following up GitHub issues here :). If I am not hearing from you for another week, I'll close this as "won't fix". Thanks!

**Comment by RasEza** [comment]
Hi, first of all I really appreciate you looking at this and sorry for not responding sooner.

My department is in the process of modernizing and migrating our legacy platform to a newer and more appropriate architecture. 

**Our platform**

The platform consists of several websites and applications that run on a series of IIS instances on Windows Servers. It serves a wide range of paying customers in the range of 50,000-500,000+. It's built almost entirely in Classic ASP (VBScript and such) on both the front and back end, without much effort put into things such as local development environments, replicable deployments, testing, you name it. 
As such it is littered with "hacks" that strongly couple it to the Windows installation it is running on, and in our case especially the installation's language configuration. A couple of instances we have run into are: 
1. Reading and writing files with special characters end up mangled due to incorrect encoding 
2. Parsing dates as strings failing
These issues may possibly to fix by themselves but due to the sheer size of the platform, it is hard to discover all such issues before they cause trouble.

**Docker use case**

Initially the wish is to containerize our legacy applications, so that we can distribute an easily reproducible and consistent environment among our developers (on Windows 10 machines). We are at a point where we have a series of working container configurations that allow for debugging and development. As a workaround to the above mentioned issues we have resorted to using a server core image with the sv-se language pack installed, but would strongly prefer to use an image with the da-DK language pack, as that is closer the platform was originally developed for (Windows Server (various editions)).

 Our current focus is primarily on improving our development proces, but with that in place we hope to expand our use of 
containers even further.

**Comment by weijuans-msft** [comment]
Wow, @RasEza, excellent background info, thank you!!

Interesting to note (correct me if I got it wrong): you are using containers on Win10 machines not Server; it's mainly in local dev environment, not on a cloud like Azure; probably no orchestrators involved; you probably care less on the container image size (hence boot up perf)? 

I wonder if you have tried this 3rd image which have much broader API support and have fonts https://hub.docker.com/_/microsoft-windows. The size is bigger though.

**Comment by RasEza** [comment]
@weijuans-msft that's right, initially the containers will only run om Win10 machines, where the size of the images dont matter as much. We may consider transitioning our current configuration to azure at some point, but that is still in the consideration stage.

> I wonder if you have tried this 3rd image which have much broader API support and have fonts https://hub.docker.com/_/microsoft-windows. The size is bigger though.

We have in fact not tried this image, thank you for bringing it to my attention! We have primarily experimented with Server Core images, but it would be great if we can achieve what we want with this. 

I'll post another comment here when we have looked into it (hopefully within a week).

**Comment by weijuans-msft** [comment]
@RasEza just checking in to see if that 3rd container image works out for you.

**Comment by RasEza** [comment]
Sorry to leave you hanging, once again thank you for following up on this issue, due to various circumstances I have not had time test the image yet.
I have more dedicated time for this issue from next week, and will of course post my results.

**Comment by weijuans-msft** [comment]
@RasEza I appreciated the update!

**Comment by weijuans-msft** [comment]
@RasEza wonder if this is a priority that you want us to look into.

**Comment by rasap** [comment]
I am so sorry that I keep delaying and postponing... We are, and I am, very invested in this issue, but other responsibilities and issues always seem to crop up... 
I know it is a lot to ask, but can you allow this issue to stay open? I can't promise a super concrete time frame, as looking into the suggested image and running through the required experimentation will require some dedicated resources from our (and my) part. I do hope that we can be back with something concrete within a reasonable time frame, but it may be up to a couple of months sadly...

By the way, this is @RasEza's corporate GitHub account. I will be responding from this one from now on.

**Comment by weijuans-msft** [comment]
@rasap @RasEza I am closing this since there is no progress for a while. Feel free to reopen if you have any update

**Comment by rasap** [comment]
@weijuans-msft im very sorry about my slow responses. I would like to provide a small update, which is that the issue seems to have been resolved by using the following image:
`mcr.microsoft.com/windows/server:win10-21h1-preview`
This image is by default configured with the correct codepage when running in Hyper-V isolation mode, 1252 and not 65001. 

Using this newer image has only been possible within the last couple of months as my organization has moved from Windows 10 Version 1809 to 21H2.

We have attempted other image variants that support 1809 with little luck, but the move to either the new image variant or to the 21H1-preview did the job.
